recipe-client-addon/lib/Storage.jsm

```diff
@@ -29,6 +30,35 @@ function loadStorage() {
 }
 
 this.Storage = {
+  DURABILITY_NAMESPACE: 'normandy_storageDurability',
+  DURABILITY_KEY: 'durable',
+  isDurabilityInvalid(value) {
+    return typeof value === 'undefined' || isNaN(value);
```

Comment:
The linter should be yelling about this, but Firefox internal code (which this add-on sort  of is) prefers double quotes.